src/EnergyPlus/UnitarySystem.hh

```diff
@@ -122,8 +185,7 @@ namespace UnitarySystems {
             UseCompressorOffFlow // set compressor OFF air flow rate equal to user defined value
         };
 
-        // friend class DesignSpecMSHP;
-
+        UnitarySysInputSpec original_input_specs;
```

Comment:
I added this as a member to UnitarySystem but never used it.

Reply:
Good cleanup, we should go in and clean out any unused member variables across the whole codebase.